source/patterns/@aws-solutions-constructs/aws-alb-lambda/lib/index.ts

```diff
@@ -158,12 +158,7 @@ export class AlbToLambda extends Construct {
 
     // If there's no listener, then we add one here
     if (newListener) {
-      this.listener = defaults.AddListener(
-        this,
-        id,
-        this.loadBalancer,
-        props.listenerProps
-      );
+      this.listener = defaults.AddListener(this, id, this.loadBalancer, props.listenerProps);
```

Comment:
You can leave this, but I'm not a fan. They are functionally equivalent, there is not a right/wrong aspect. I'm pretty sure the existing format was applied by our autoformatting tool, so will probably reappear in the future when someone says "Format Document"